SConstruct

```diff
@@ -151,6 +152,8 @@ opts.Add('platform', "Target platform (%s)" % ('|'.join(platform_list), ), '')
 opts.Add(EnumVariable('target', "Compilation target", 'debug', ('debug', 'release_debug', 'release')))
 opts.Add(BoolVariable('tools', "Build the tools a.k.a. the Godot editor", True))
 opts.Add(BoolVariable('use_lto', 'Use linking time optimization', False))
+opts.Add('msvc_version', 'MSVC version to use. Ignored if VCINSTALLDIR is set in shell env.', None)
+opts.Add(BoolVariable('use_mingw', 'Use the Mingw compiler, even if MSVC is installed. Only used on Windows.', False))
```

Comment:
Those two Windows-specific options should be added to `platform/windows/detect.py` (and likely duplicated in `platform/uwp/detect.py` I guess).

Reply:
Makes sense. Pushed a new version with this.